ci/build.sh

```diff
@@ -13,6 +13,7 @@ echo --- Build
 
   ci/docker-run.sh solanalabs/sgxsdk ./src/sgx-ecc-ed25519/build.sh
   ci/docker-run.sh solanalabs/sgxsdk ./src/sgx/signing/build.sh
```

Comment:
maybe a top-level ./src/sgx/build.sh that builds and tests?

Reply:
👍